mail-integration-api/src/main/java/org/exoplatform/mailintegration/model/MailIntegrationUserSetting.java

```diff
@@ -29,5 +29,5 @@
   
```

Comment:
MailIntegrationUserSetting should be removed since there will be no corresponding entity